Source/WebCore/platform/graphics/skia/PathSkia.cpp

```diff
@@ -164,9 +150,20 @@ void PathSkia::add(PathClosedArc closedArc)
     add(PathCloseSubpath());
 }
 
-void PathSkia::add(PathEllipse)
+void PathSkia::add(PathEllipse ellipse)
 {
-    notImplemented();
+    if (ellipse.rotation) {
```

Comment:
I personally would prefer to move the m_elementStream->add(ellipse) to the top, then

```
if (!ellipse.rotation) {
    addEllipse(...)
    return;
}

auto transform = AffineTransform::makeTranslation(...);
```

to avoid indenting the long branch.

Reply:
I don't know what the element stream is, is it ok to call that before adding the platform ellipse?